rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import typescript from 'rollup-plugin-typescript2';
+import { terser } from "rollup-plugin-terser";
+import * as path from "path";
+
+const BASE_PATH = process.cwd();
+const getFile = path => path.resolve(BASE_PATH, path);
+
+const pkg = require(getFile('package.json'));
+
+export default {
+  input: path.resolve(BASE_PATH, 'src/index.ts'),
+  external: Object.keys({ ...pkg.dependencies, ...pkg.peerDependencies }),
+  plugins: [
+    typescript({
+      tsconfig: getFile('tsconfig.build.json'),
```

Comment:
Where's this file supposed to be coming from? Most of our packages do not have it